frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/properties/basicTicketImage/ticketImageAction/actions/actions.component.tsx

```diff
@@ -0,0 +1,127 @@
+/**
+ *  Copyright (C) 2022 3D Repo Ltd
+ *
+ *  This program is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Affero General Public License as
+ *  published by the Free Software Foundation, either version 3 of the
+ *  License, or (at your option) any later version.
+ *
+ *  This program is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Affero General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Affero General Public License
+ *  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import GoToViewpoinIcon from '@assets/icons/outlined/target-outlined.svg';
+import AddViewpointIcon from '@assets/icons/outlined/add_circle-outlined.svg';
+import ChangeViewpointIcon from '@assets/icons/outlined/rotate_arrow-outlined.svg';
+import { FormattedMessage } from 'react-intl';
+import { ActionMenuItem } from '@controls/actionMenu/actionMenuItem/actionMenuItem.component';
+import AddImageIcon from '@assets/icons/outlined/add_image-outlined.svg';
+import EditImageIcon from '@assets/icons/outlined/edit-outlined.svg';
+import { useContext, useRef } from 'react';
+import { Viewer as ViewerService } from '@/v4/services/viewer/viewer';
+import { getFileFromInputEvent } from '@controls/formImage/image.helper';
+import { HiddenImageInput } from '@controls/formImage/hiddenImageUploader/hiddenImageUploader.styles';
+import { HiddenInputContainer } from '@controls/formImage/hiddenFileUploader/hiddenFileUploader.styles';
+import { ActionMenu, MenuItem, MenuItemDelete } from './actions.styles';
+import { TicketImageActionContext } from '../ticketImageActionContext';
+import { TicketImageAction } from '../ticketImageAction.component';
+
+export const GoToViewpointAction = () => (
+	<TicketImageAction disabled={false}>
+		<GoToViewpoinIcon />
+		<FormattedMessage id="viewer.card.ticket.viewpoint.action.goToViewpoint" defaultMessage="Go to viewpoint" />
```

Comment:
Should the id follow the same path as other messages in this file (i.e. ticketImage not ticket)